integration_tests/ads/test_conversion_events.py

```diff
@@ -8,6 +8,8 @@
 from pinterest.client import PinterestSDKClient
 from pinterest.ads.conversion_events import Conversion
 
+from openapi_generated.pinterest_client.exceptions import ApiException
```

Comment:
Why are we importing this? I do not see any uses of it in the file. If we need one, there is an `SDKException` class defined, please use that instead.